index.html

```diff
@@ -6,59 +6,244 @@
+        <nav>
+            <ul class="header-nav-desktop">
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
+            </ul>
+        </nav>
+        <div class="container container-flex">
+            <h1 class="header-title">
+                Hey there. I’m Shingirai
+                <span>I’m a software developer</span>
+            </h1>
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
 
-        <ul class="social-media">
-            <li>
-                <a href="">
-                    <img src="./assets/social-media/twitter.svg" alt="twitter icon">
-                </a>
-            </li>
-            <li>
-                <a href="">
-                    <img src="./assets/social-media/linkedin.svg" alt="linkedin icon">
-                </a>
-            </li>
+            <ul class="social-media">
+                <li>
+                    <a href="">
+                        <img src="./assets/social-media/twitter.svg" alt="twitter icon">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="./assets/social-media/linkedin.svg" alt="linkedin icon">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="./assets/social-media/medium.svg" alt="medium icon">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="./assets/social-media/github.svg" alt="github icon">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="./assets/social-media/angellist.svg" alt="angellist icon">
+                    </a>
+                </li>
+            </ul>
+
+            <button type="button">
+                <img src="./assets/Disabled.svg" alt="disabled button">
+            </button>
+        </div>
+    </section>
+    <section>
+        <div class="works">
+            <h1>My Recent Works</h1>
+            <article class="card-work">
+                <div class="card-image">
+                    <img src="./assets/project1.png" alt="image work">
+                </div>
+                <h2 class="card-title">
+                    Tonic
+                </h2>
+                <ul class="card-tags">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button type="button">
+                 See Project   
+                </button>
+            </article>
+            <article class="card-work">
```

Comment:
- [ ] Please we don't use the `article` element  as a styling wrapper is  this non-semantic and not a good practice
Please use div instead